script/worker-start

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+set -x
+
+# make sure our pids dir exists
+mkdir -p tmp/pids
```

Comment:
this is used on cloud.gov, where we do not run `bin/setup`. It's fine to have it in both places.